pkg/minikube/console/style.go

```diff
@@ -47,67 +46,67 @@ type style struct {
-	"crushed":       {Prefix: "💔  "},
-	"url":           {Prefix: "👉  ", LowPrefix: lowIndent},
-	"documentation": {Prefix: "📘  "},
-	"issues":        {Prefix: "⁉️   "},
-	"issue":         {Prefix: "    ▪ ", LowPrefix: lowIndent}, // Indented bullet
-	"check":         {Prefix: "✔️  "},
+var styles = map[StyleEnum]style{
+	Happy:         {Prefix: "😄  "},
+	SuccessType:   {Prefix: "✅  "},
+	FailureType:   {Prefix: "❌  "},
+	Conflict:      {Prefix: "💥  ", LowPrefix: lowWarning},
+	FatalType:     {Prefix: "💣  ", LowPrefix: lowError},
+	Notice:        {Prefix: "📌  "},
+	Ready:         {Prefix: "🏄  "},
+	Running:       {Prefix: "🏃  "},
+	Provisioning:  {Prefix: "🌱  "},
+	Restarting:    {Prefix: "🔄  "},
+	Reconfiguring: {Prefix: "📯  "},
+	Stopping:      {Prefix: "✋  "},
+	Stopped:       {Prefix: "🛑  "},
+	WarningType:   {Prefix: "⚠️  ", LowPrefix: lowWarning},
+	Waiting:       {Prefix: "⌛  "},
+	WaitingPods:   {Prefix: "⌛  ", OmitNewline: true},
+	Usage:         {Prefix: "💡  "},
+	Launch:        {Prefix: "🚀  "},
+	Sad:           {Prefix: "😿  "},
+	ThumbsUp:      {Prefix: "👍  "},
+	Option:        {Prefix: "    ▪ ", LowPrefix: lowIndent}, // Indented bullet
+	Command:       {Prefix: "    ▪ ", LowPrefix: lowIndent}, // Indented bullet
+	LogEntry:      {Prefix: "    "},                         // Indent
+	Crushed:       {Prefix: "💔  "},
+	URL:           {Prefix: "👉  ", LowPrefix: lowIndent},
+	Documentation: {Prefix: "📘  "},
+	Issues:        {Prefix: "⁉️   "},
+	Issue:         {Prefix: "    ▪ ", LowPrefix: lowIndent}, // Indented bullet
+	Check:         {Prefix: "✔️  "},
 
 	// Specialized purpose styles
-	"iso-download":      {Prefix: "💿  "},
-	"file-download":     {Prefix: "💾  "},
-	"caching":           {Prefix: "🤹  "},
-	"starting-vm":       {Prefix: "🔥  "},
-	"starting-none":     {Prefix: "🤹  "},
-	"resetting":         {Prefix: "🔄  "},
-	"deleting-host":     {Prefix: "🔥  "},
-	"copying":           {Prefix: "✨  "},
-	"connectivity":      {Prefix: "📶  "},
-	"internet":          {Prefix: "🌐  "},
-	"mounting":          {Prefix: "📁  "},
-	"celebrate":         {Prefix: "🎉  "},
-	"container-runtime": {Prefix: "🎁  "},
-	"Docker":            {Prefix: "🐳  "},
-	"CRI-O":             {Prefix: "🎁  "}, // This should be a snow-flake, but the emoji has a strange width on macOS
-	"containerd":        {Prefix: "📦  "},
-	"permissions":       {Prefix: "🔑  "},
-	"enabling":          {Prefix: "🔌  "},
-	"shutdown":          {Prefix: "🛑  "},
-	"pulling":           {Prefix: "🚜  "},
-	"verifying":         {Prefix: "🤔  "},
-	"verifying-noline":  {Prefix: "🤔  ", OmitNewline: true},
-	"kubectl":           {Prefix: "💗  "},
-	"meh":               {Prefix: "🙄  ", LowPrefix: lowWarning},
-	"embarrassed":       {Prefix: "🤦  ", LowPrefix: lowWarning},
-	"tip":               {Prefix: "💡  "},
-	"unmount":           {Prefix: "🔥  "},
-	"mount-options":     {Prefix: "💾  "},
-	"fileserver":        {Prefix: "🚀  ", OmitNewline: true},
+	IsoDownload:      {Prefix: "💿  "},
```

Comment:
This should be ISODownload: https://github.com/golang/go/wiki/CodeReviewComments#initialisms